docs/CHANGELOG.asciidoc

```diff
@@ -526,6 +527,44 @@ Search::
 * Fixes an issue in Search Playground where context limit errors were not handled well when using the Elastic Managed LLM ({kibana-pull}225360[#225360]).
 * Adjusts the `z-index` of the app menu header to not conflict with the Persistent Console ({kibana-pull}224708[#224708]).
 
+
+[[release-notes-8.18.5]]
+== {kib} 8.18.5
+
+The 8.18.5 release includes the following fixes.
+
+[float]
+[[fixes-v8.18.5]]
+=== Fixes
+Dashboards and Visualizations::
+* Fixes an issue with the plus sign `+` not being properly encoded when it's part of a date math expression ({kibana-pull}230469[#230469]).
+* Fixes an issue in Firefox preventing from scrolling in the Lens editor flyout ({kibana-pull}228625[#228625]).
+* Fixes an issue preventing from saving linked TSVB visualizations when changing data view ({kibana-pull}228685[#228685]).
```

Comment:
```suggestion
* Fixes an issue preventing you from saving linked TSVB visualizations when changing data views ({kibana-pull}228685[#228685]).
```

Reply:
Idk which is preferred. But linking so @florent-leborgne can sync things up.